src/db/client.js

```diff
@@ -120,6 +121,11 @@ async function listTables(server, database) {
   return database.connection.listTables(database.database);
 }
 
+async function listSchemas(server, database) {
```

Comment:
Ok i'll change that ... i did it because all other list*() have async :)
like listRoutines or listTableColumns